### DISCUSSION TITLE
`[@types/i18next-fs-backend]` not being published by the bot?

### DISCUSSION THREAD

**Discussion by favna** [DefinitelyTyped/DefinitelyTyped#55988] [Issues with a @types package] [answered]
Hello there,

A while ago 3 PRs were created between myself and a friend of mine for [@types/i18next-fs-backend](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/i18next-fs-backend) but none are being published. What's up with this? We need the updated types for a project before we can release it.

PR 1: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55876
PR 2: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55893
PR 3: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55926

[a comment by an automated account is not shown]

**Reply by favna** [reply]
Going to add ping of a DT maintainer here because (obvioussly) Quantumlyy nor I can fix this

@peterblazejewicz

**Comment by peterblazejewicz** [comment, marked as answer]
types are listed on NPM but now content, is this what you're seeing?
https://www.npmjs.com/package/@types/i18next-fs-backend

**Reply by peterblazejewicz** [reply]
I'm getting exact content listed here (same version):
```bash
 npm pack @types/i18next-fs-backend
npm notice
npm notice package: @types/i18next-fs-backend@1.1.2
npm notice === Tarball Contents ===
npm notice 1.1kB i18next-fs-backend/LICENSE
npm notice 2.5kB i18next-fs-backend/README.md
npm notice 1.7kB i18next-fs-backend/index.d.ts
npm notice 1.0kB i18next-fs-backend/package.json
npm notice === Tarball Details ===
npm notice name:          @types/i18next-fs-backend
npm notice version:       1.1.2
npm notice filename:      types-i18next-fs-backend-1.1.2.tgz
npm notice package size:  2.0 kB
npm notice unpacked size: 6.3 kB
npm notice shasum:        4f3116769229371fcdf64bbdb6841ea745e392f9
npm notice integrity:     sha512-ZzTRXA5B0x0oG[...]z9O6CjPrAlmIQ==
npm notice total files:   4
npm notice
types-i18next-fs-backend-1.1.2.tgz
```

**Reply by favna** [reply]
Well that's totally bizarre. I always use Yarn and when I go to their website (https://yarnpkg.com/package/@types/i18next-fs-backend) it shows latest being 1.0.1. But then when I execute `yarn info @types/i18next-fs-backend` it shows latest is `1.1.2`. And yeah I can confirm that NPM also shows 1.1.2. 

I also don't know what's up with NPM because when I created this discussion earlier today I did check NPM and it too said 1.0.1. 

Content for 1.1.2 is correct so I guess I'll mark this as the answer.
